cg/meta/workflow/nf_analysis.py

```diff
@@ -183,22 +254,16 @@ def verify_deliverables_file_exists(self, case_id: str) -> None:
         if not Path(self.get_deliverables_file_path(case_id=case_id)).exists():
             raise CgError(f"No deliverables file found for case {case_id}")
 
-    def write_params_file(self, case_id: str, workflow_parameters: dict) -> None:
+    def write_params_file(self, case_id: str, workflow_parameters: dict = None) -> None:
         """Write params-file for analysis."""
         LOG.debug("Writing parameters file")
-        write_yaml_nextflow_style(
-            content=workflow_parameters,
-            file_path=self.get_params_file_path(case_id=case_id),
-        )
-
-    def write_nextflow_config(self, case_id: str) -> None:
-        """Write nextflow config in json format."""
-        if content := self.get_nextflow_config_content():
-            LOG.debug("Writing nextflow config file")
-            write_txt(
-                content=content,
-                file_path=self.get_nextflow_config_path(case_id=case_id),
+        if workflow_parameters:
+            write_yaml_nextflow_style(
+                content=workflow_parameters,
+                file_path=self.get_params_file_path(case_id=case_id),
             )
+        else:
+            self.get_params_file_path(case_id=case_id).touch()
```

Comment:
```suggestion
        if not workflow_parameters:
            self.get_params_file_path(case_id=case_id).touch()
        write_yaml_nextflow_style(
                content=workflow_parameters,
                file_path=self.get_params_file_path(case_id=case_id),
            )

```

Reply:
removes an else, subjective though

Reply:
I will keep as it is, seems clearer to me since the `touch` part is a bit special and only done in certain cases